doc/developer/tooling.rst

```diff
@@ -115,15 +115,14 @@ only use third-party Python libraries that have MIT or BSD licenses.
 
 Releasing a New Version of ARMI
 -------------------------------
-In ARMI, we use the common ``major.minor.bump`` version scheme where a version string
-might look like ``0.1.7``, ``1.0.0``, or ``1.2.123``. Each number has a specific meaning:
+We use the common ``major.minor.bump`` version scheme where a version string
+might look like ``0.1.7``, ``1.0.0``, or ``12.3.123``. Each number has a specific meaning:
 
-* ``major`` - Revved for NRC-sanctioned release or at the end of a long development cycle.
-* ``minor`` - Revved when we decide the code or our API has reached a stable point;
-  this might happen once a year.
-* ``bump`` - Revved every time we modify the API, or at will; any time we want.
+* ``major`` - Revved for DOE/NRC-sanctioned release or at the end of a long development cycle.
+* ``minor`` - Revved when we decide the code or our API has reached a stable point.
+* ``bump`` - Revved every time we modify the API, or at any time we want.
```

Comment:
As the code base matures, I think it makes more sense to have the API changes rev the version in a more substantial way. For now, there are still enough alterations to the API that it doesn't make sense to have it cause a major bump, but at least a minor. I also don't think the `minor` guidance makes much sense in general (at least to me). 

So here are my suggestions:

1. minor: API changes + large feature changes/additions
2. micro/bump: smaller feature changes/additions, or after a period of time with many updates

Reply:
> 2. micro/bump: smaller feature changes/additions

I'm not sure about that. That would mean we rev the `bump` version on 50% of our PRs.  I'll have to think about it, anyway. hmm

Reply:
I guess large and small aren't descriptive and are relative. I certainly don't think it makes sense to bump on 50% of PRs, then versions lose their meaning. If I'm thinking about the last ~6 months with this scheme I've proposed _not including API changes_, there have been a few large overhauls (3-4?) that should have gotten a minor rev, and maybe ...hmmm.... 10-15 micro revs??? Hard to say since I'm not as involved with PRs as you, but that's the frequency I have in my mind.

Reply:
Arrielle, I have not dropped this topic. Sorry, I just got distracted. I'm looking at it now.